components/Opaque.js

```diff
@@ -0,0 +1,4 @@
+const Opaque = (props) => {
```

Comment:
For optimization reasons, it's not going to be a good idea to spread out little abstract components like this.

We can still do this, but I would collect them in a single file.